Tests/FluentAssertions.Specs/Specialized/TaskOfTAssertionSpecs.cs

```diff
@@ -2,13 +2,11 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Threading.Tasks;
 using FluentAssertions.Extensions;
-#if NET47
```

Comment:
I'm pretty sure this is still used for `Divide` and `Multiply` extension methods.

Reply:
Sorry for that!
Wondering why this was not seen before in the local build?
I trusted to much in Rider...

Reply:
No worries, I have *almost* removed it several times.

Reply:
How about moving it to top level namespace (in Specs) to prevent this problem in future?

Reply:
I've added a note to myself about this.
It's been a bit back and forth with that class
https://github.com/fluentassertions/fluentassertions/pull/1376/files
https://github.com/fluentassertions/fluentassertions/pull/1424/files